### PR TITLE
Step 4

### PR DESCRIPTION
**Pull request by EugeneTeu**
- add extra tab page for sps content 
- add post form for user to add comment to server

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java [hunk 1]
```diff
@@ -51,4 +43,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "user-comment", "");
+    if (text.isEmpty()) {
+      response.setContentType("text/html");
+      response.getWriter().println("Please enter comment");
+      return;
+    }
+    comments.add(text);
+    // Redirect back to the HTML page.
```
Comment (fanwgwg) [inline portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:68]: nit: Redirect back to home page?

File: portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java [hunk 2]
```diff
@@ -51,4 +43,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+      return defaultValue;
+    }
+    return value;
+  }
+
+  @Override 
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "user-comment", "");
+    if (text.isEmpty()) {
```
Comment (fanwgwg) [inline portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:62]: Optional: do you want to perform some input validation? E.g, if user posts a list of whitespaces, shall we ignore or still store and show them?

Comment (EugeneTeu) [reply portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:62]: will add in the next pr

File: portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java (hunk 1 above)
Comment (EugeneTeu) [reply portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:68]: will change in next pr!

**Pull request merged**